integrations/spark/spark-validate-cleanse/spark-validate-cleanse-app/src/main/java/com/thinkbiganalytics/spark/datavalidator/Validator.java

```diff
@@ -45,57 +45,57 @@ public static void main(String[] args) {
         if (log.isInfoEnabled()) {
             log.info("Running Spark Validator with the following command line args (comma separated):{}", StringUtils.join(args, ","));
         }
-
-        try {
-            final int result = new Validator().run(System.out, args);
-            System.exit(result);
-        } catch (final Exception e) {
-            log.error("Failed to perform validation: ", e.toString(), e);
-            System.out.println(e);
-            System.exit(1);
-        }
+        new Validator().run(System.out, args);
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Replace this use of System.out or System.err by a logger. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS106)